jbpm-examples/src/main/resources/datasource.properties

```diff
@@ -0,0 +1,23 @@
+# for creating a datasource
+driverClassName=org.h2.Driver
+className=org.h2.jdbcx.JdbcDataSource
+user=sa
+password=
+url=jdbc:h2:tcp://localhost/~/jbpm-db
+datasourceName=jdbc/jbpm-ds
```

Comment:
You can leave only these properties here, others should not be used by PersistenceUtil class. They are only used by current jBPMHelper class.

Reply:
These two, task service are used by ProcessMain which uses JBPMHelper. See additional comments below.

taskservice.enabled=true
taskservice.datasource.name=org.jbpm.services.task